src/com/epam/javatrainings/annootation_based_DTO/processor/LengthAnnotationProcessor.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.javatrainings.annootation_based_DTO.processor;
+
+import com.epam.javatrainings.annootation_based_DTO.persistance.annotation.Length;
+import com.epam.javatrainings.annootation_based_DTO.persistance.model.CustomerDto;
+
+import java.lang.reflect.Field;
+
+public class LengthAnnotationProcessor {
+    public static boolean checkValidateLength(CustomerDto customerDto) throws NoSuchFieldException, IllegalAccessException {

```

Comment:
you can handle exceptions in this layer and throw your custom exception with necessary info (e.g. fieldName, errorMessage) or directly return that info as list of errors.